plugins/org.jkiss.dbeaver.ext.firebird/src/org/jkiss/dbeaver/ext/firebird/model/FireBirdSQLDialect.java

```diff
@@ -44,11 +44,26 @@ public class FireBirdSQLDialect extends GenericSQLDialect {
     };
 
     private static final String[] FIREBIRD_KEYWORDS = new String[] {
+        "ACCENT",
+        "BLOCK",
+        "BREAK",
         "COMMENT",
+        "COMPUTED",
+        "CONTAINING",
         "CURRENT_USER",
         "CURRENT_ROLE",
+        "GENERATOR",
         "NCHAR",
-        "VALUE"
+        "STARTING",
+        "VALUE",
+        "WEEKDAY",
+        "YEARDAY",
+    };
+
+    private static final String[] FIREBIRD_FUNCTIONS = {
```

Comment:
Okay, but why only 3? 
Did you check other functions https://firebirdsql.org/refdocs/langrefupd25-intfunc.html?

Reply:
I only added these three because functions were not my goal. But I can do it.

Reply:
Please, do it.

Reply:
It's done.